src/NServiceBus.AcceptanceTests/Recoverability/Retries/When_custom_policy_always_moves_to_error.cs

```diff
@@ -37,15 +37,11 @@ public RetryEndpoint()
                 {
                     var scenarioContext = (Context) context.ScenarioContext;
                     configure.EnableFeature<TimeoutManager>();
-                    configure.SecondLevelRetries().CustomRetryPolicy(RetryPolicy);
+                    configure.Recoverability()
+                        .CustomPolicy((cfg, errorContext) => RecoverabilityAction.MoveToError());
                     configure.Notifications.Errors.MessageSentToErrorQueue += (sender, message) => { scenarioContext.MessageSentToErrorQueue = true; };
                 })
-                    .WithConfig<SecondLevelRetriesConfig>(c => c.TimeIncrease = TimeSpan.FromMilliseconds(1));
-            }
-
-            static TimeSpan RetryPolicy(SecondLevelRetryContext slrRetryContext)
-            {
-                return TimeSpan.MinValue;
+                .WithConfig<SecondLevelRetriesConfig>(c => c.TimeIncrease = TimeSpan.FromMilliseconds(1));
```

Comment:
I would rather remove it. Otherwise we get a test that has unclear focus and we test passing the configuration with `When_custom_policy_provided`